src/win/getaddrinfo.c

```diff
@@ -261,7 +276,8 @@ int uv_getaddrinfo(uv_loop_t* loop,
 
   /* calculate required memory size for all input values */
   if (node != NULL) {
-    nodesize = ALIGNED_SIZE(uv_utf8_to_utf16(node, NULL, 0) * sizeof(WCHAR));
+    nodesize = ALIGNED_SIZE(MultiByteToWideChar(CP_UTF8, 0, node, -1, NULL, 0) *
+                            sizeof(WCHAR));
```

Comment:
This one seems to fall into the same category, the `sizeof(WCHAR)` isn't part of `MultiByteToWideChar()`.